modules/ROOT/pages/vpc-firewall-rules-concept.adoc

```diff
@@ -31,9 +31,9 @@ When creating a VPC, 4 firewall rules are created by default:
 },
 ----
 +
-These firewall rules allow traffic from the VPC to reach your workers through ports 8091 and 8092. These are the only ports used by your CloudHub dedicated load balancer to proxy all external communications to your workers through.
+These firewall rules allow traffic from within the VPC to reach your workers through ports 8091 and 8092. These are the only ports used by your CloudHub dedicated load balancer to proxy all external communications to your workers.
```

Comment:
since CloudHub dedicated modifies balancer, should it be hyphenated? CloudHub-dedicated load balancer?